go.mod

```diff
@@ -11,3 +11,5 @@ require (
 	github.com/ipld/go-ipld-prime v0.16.0
 	github.com/ipld/go-ipld-prime/storage/bsadapter v0.0.0-20211210234204-ce2a1c70cd73
 )
+
+replace github.com/ipld/go-car/v2 => ../go-car/v2
```

Comment:
@willscott  ?

Reply:
i can link it against https://github.com/ipld/go-car/pull/305 if you car, but was developing the two together. this code demonstration ended up as a test case in that go-car PR